cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2021-08-14T00:00:00Z
+-- index-state: 2021-08-14T00:00:00Z
```

Comment:
Can we not just update this to a more recent time over removing it?

Reply:
Actually we don't even need to bump the date, I just tried to apply Las's suggestion and forgot to revert.

Reply:
Reverted now.